deployment/hasura/metadata/actions.graphql

```diff
@@ -291,29 +291,33 @@ type ExpandedActivityInstance {
 type SequenceSeqJson {
   id: String!
   metadata: Any!
-  steps: [CommandSeqJson!]!
+  steps: [CommandSeqJson!]
```

Comment:
Please do not merge until after the release

Reply:
@camargo did you have a question about this change? Steps are not required anymore, but if an array is provided it needs to have `CommandSeqJson`

Let me know if that above captures that intent

Reply:
Ah sorry, did not mean to put the review here! Looks good to me.